packages/framer-motion/src/render/store.ts

```diff
@@ -0,0 +1,3 @@
+import type { VisualElement } from "./VisualElement"
+
+export const visualElementStore = new WeakMap<any, VisualElement>()
```

Comment:
Perhaps for readability:
```suggestion
type Instance = object // or Element?
export const visualElementStore = new WeakMap<Instance, VisualElement>()
```